e2e/clusterctl-config.yaml

```diff
@@ -0,0 +1,16 @@
+providers:
+  - name: local-linode
+    url: /home/runner/work/cluster-api-provider-linode/cluster-api-provider-linode/infrastructure-local-linode/v0.0.0/infrastructure-components.yaml
+    type: InfrastructureProvider
+  - name: "k3s"
+    url: https://github.com/k3s-io/cluster-api-k3s/releases/latest/bootstrap-components.yaml
+    type: "BootstrapProvider"
+  - name: "k3s"
+    url: https://github.com/k3s-io/cluster-api-k3s/releases/latest/control-plane-components.yaml
+    type: "ControlPlaneProvider"
+  - name: "rke2"
```

Comment:
This should be built into clusterctl, I think we can remove it from here

Reply:
I originally didn't have it but for some reason, it was throwing errors when the `clusterctl generate` command was executed. Locally didn't have that issue...

Reply:
maybe we need a newer clusterctl version, we can leave it for now